src/AWS.Deploy.Orchestration/DeploymentBundleHandler.cs

```diff
@@ -56,6 +56,9 @@ public async Task<string> BuildDockerImage(CloudApplication cloudApplication, Re
             var buildArgs = GetDockerBuildArgs(recommendation);
 
             var dockerBuildCommand = $"docker build -t {imageTag} -f \"{dockerFile}\"{buildArgs} .";
+            _interactiveService.LogMessageLine($"Docker Execution Directory: {Path.GetFullPath(dockerExecutionDirectory)}");
+            _interactiveService.LogMessageLine($"Docker Build Command: {dockerBuildCommand}");
```

Comment:
Can we use LogDebugLine instead? I'd rather we only show this info when we are in diagnostics mode.

Reply:
I would rather keep it as a regular logging message. I have found this information in the output really helpful when debugging issues in our other CLIs. Users who have problems often put this output as part of the issue.